test/decl/inherit/initializer.swift

```diff
@@ -110,3 +110,17 @@ func testClassInGenericFunc<T>(t: T) {
   _ = B(t: t)
 }
 
+
+// <https://bugs.swift.org/browse/SR-5056> Required convenience init inhibits inheritance
+
+class SR5056A {
+    required init(a: Int) {}
+}
+
+class SR5056B : SR5056A {
+    required convenience init(b: Int) {
+        self.init(a: b)
+    }
+}
+
+class SR5056C : SR5056B {}
```

Comment:
Can you add a function that calls both initializers on C, just to prove that they are still present after all?

Reply:
Sure.